Gifski/DraggableFile.swift

```diff
@@ -13,8 +13,17 @@ final class DraggableFile: NSImageView {
 		super.init(frame: frame)
 		isEditable = false
 		unregisterDraggedTypes()
+		
+		superview?.wantsLayer = true
+		wantsLayer = true
+		
+		let sh = NSShadow()
```

Comment:
You can use the `with()` function in `util.swift` to simplify the construction of this.